scrapi/base/helpers.py

```diff
@@ -2,11 +2,24 @@
 
 import re
 from copy import deepcopy
+import functools
+
 from nameparser import HumanName
 
 URL_REGEX = re.compile(ur'(https?://\S*\.\S*)')
 
 
+def single_result(l, default=''):
+    return l[0] if len(l) == 1 else default
```

Comment:
Wasn't sure what the behavior should be if there was more than 1 result. Should it error, or just return the first element in the list?